code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -18,6 +18,9 @@
 			if((D.spread_flags & DISEASE_SPREAD_SPECIAL) || (D.spread_flags & DISEASE_SPREAD_NON_CONTAGIOUS))
 				continue
 
+			if ( ( data[ "donor" ] && istype( data[ "donor" ], /mob/living/simple_animal/mouse ) ) && islizard( L ) )
```

Comment:
shouldn't this check their "can eat mice okay" status instead of their lizard species?

Reply:
Races that can eat mice without being disgusted, I also believe it is a trait

Reply:
some races enjoy mice via the liked_food = GROSS 
while others do the same via disliked_food = NONE